### ISSUE TITLE
Artisan --namespace option

### ISSUE THREAD

**Issue by thearyanahmed** [laravel/ideas#2187]
I'm currency working on a project that uses multiple modules. Which are situated outside the "app" directory, each module have its own directory and its own namespace. 
Now, when creating a controller with ```php artisan make:controller SomeController``` , it will always create in the **App** namespace. 

So, for the modules, I can not use the artisan command. 
Proposing a --namespace flag to be allowed with the ```php artisan make:*``` command. That will create the controller/other files in that namespace

```
php artisan make:controller PaymentController --namespace=Payment 
```
```
php artisan make:middleware SomeMiddleware --namespace=SomeNamespace 
```

Proposing this **--namespace**  flag to all **make:**  commands.

**Comment by alexbowers** [comment]
You can already do this, all you do instead of specifying a `--namespace` is you follow a directory structure for your classes.

Eg.

php artisan make:controller Payment/PaymentController

**Comment by thearyanahmed** [comment]
Not really, using the directory structured way will create in inside the "app" directory, but in the projects, I'm working on at the moment, our modules are outside of the "app" project,

directory structure:

```
app
bootstrap
config
database
...
modules
  moduleA
    Http
    Jobs
  moduleB
    Http
    Events
    Listeners

```

So in that case, even using Absolute path will not work.
